From 960ae1e36cf0a822d6af6362df6fcd87b899ed30 Mon Sep 17 00:00:00 2001
From: Jagdish Gediya <jagdish.gediya@nxp.com>
Date: Thu, 9 Aug 2018 00:07:16 +0530
Subject: [PATCH 082/107] armv8: icid: fix 'set_icids' call

'set_icids' is defined only for CONFIG_ARCH_LS1043A and
CONFIG_ARCH_LS1046A.

call set_icids only if CONFIG_ARCH_LS1043A or CONFIG_ARCH_LS1046A
is defined.

Signed-off-by: Jagdish Gediya <jagdish.gediya@nxp.com>
---
 arch/arm/cpu/armv8/fsl-layerscape/soc.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/cpu/armv8/fsl-layerscape/soc.c b/arch/arm/cpu/armv8/fsl-layerscape/soc.c
index 7814096..5f00d3c 100644
--- a/arch/arm/cpu/armv8/fsl-layerscape/soc.c
+++ b/arch/arm/cpu/armv8/fsl-layerscape/soc.c
@@ -688,7 +688,9 @@ void fsl_lsch2_early_init_f(void)
 	erratum_a008997();
 	erratum_a009007();
 
+#if defined(CONFIG_ARCH_LS1043A) || defined(CONFIG_ARCH_LS1046A)
 	set_icids();
+#endif
 }
 #endif
 
-- 
1.7.1

